webserver/k8s/crd.yaml

```diff
@@ -1,23 +1,35 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This should be generated automatically using the CRD generator.

Reply:
I tested it for websierver at least, today doing a presentation on CNCF meetup, it looked ugly that we still use the old beta api.

Well not sure we want ot have it in every example, users's not necessary want to generate yaml  I guess. But we can make at least one example that way in future.

Reply:
I tested it for websierver at least, today doing a presentation on CNCF meetup, it looked ugly that we still use the old beta api.

Well not sure we want ot have it in every example, users's not necessary want to generate yaml I guess. But we can make at least one example that way in future.